docs/instructor-guide/index.md

```diff
@@ -0,0 +1,26 @@
+# Instructor Guide
+
+This section provides detailed documentation for all aspects of using PrairieLearn as an instructor.
+
+To develop questions and other course content on your own computer, see the [local installation guide](../installing.md).
+
+Learn more about configuring the four main aspects of your course:
+
+- [Courses](../course/index.md)
+- [Course instances](../courseInstance.md)
+- [Assessments](../assessment/index.md)
+- [Questions](../question.md)
+
+!!! tip
+
+    For a refresher of what these aspects are, see the [PrairieLearn concepts](../concepts/index.md).
+
+Some other important links are:
+
+- [Access control](../accessControl/index.md): learn how to control when and where students can access your course content.
+- [Element catalog](../elements.md): a reference for all elements available for use in questions
+- [Code autograders](../externalGrading.md): learn about the external code autograders
+  - [Python](../python-grader/index.md)
+  - [C / C++](../c-grader/index.md)
+- [FAQ](../faq.md): Common questions and answers
+- [Editing and syncing course content](../sync.md): Tradeoffs between in-browser editing and local editing
```

Comment:
Capitalization/punctuation consistency nits. Also includes a reference to the Java grader.

Maybe include links to various bits of workspace documentation too? Those are a pretty fundamental PL feature and differentiator.

```suggestion
- [Element catalog](../elements.md): a reference for all elements available for use in questions.
- [Code autograders](../externalGrading.md): learn how to automatically grade student code.
  - [Python](../python-grader/index.md)
  - [C / C++](../c-grader/index.md)
  - [Java](../java-grader/index.md)
- [FAQ](../faq.md): find answers to common questions.
- [Editing and syncing course content](../sync.md): understand tradeoffs between in-browser editing and local editing.
```